spec/controllers/answers_controller_spec.rb

```diff
@@ -46,6 +46,34 @@
     end
   end
 
+  describe "PATCH#approve" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.